integration_tests/tests/s3.rs

```diff
@@ -227,6 +228,61 @@ fn test_multipart_upload(client: &TestClient, bucket: &str, filename: &str) {
     let response = client.complete_multipart_upload(complete_req).sync().expect("Couldn't complete multipart upload");
     println!("{:#?}", response);
 
+    // Add copy upload part to this test
+    // https://docs.aws.amazon.com/AmazonS3/latest/API/mpUploadUploadPartCopy.html
+    let create_multipart_req2 = CreateMultipartUploadRequest {
```

Comment:
Looks like the Ceph integration tests we run on SemaphoreCI fails here. Perhaps Ceph doesn't support this particular command. First thing to do would be see what the response body is they return. Logs are in Semaphore CI, click the `Job #1` then `.semaphoreci/run.sh` to unroll the logs there. 😄

Reply:
@matthewkmayer on it now! Totally didn't realise there were tests I wasn't running :p

Reply:
It appears to be receiving a 411 because the content-length is not set... digging into how to work around that, since I'm not sure why it cares.

```
<?xml version="1.0" encoding="UTF-8"?>
Error>
    <Code>MissingContentLength</Code>
    <BucketName>rusoto-test-bucket-1542142003</BucketName>
    <RequestId>tx00000000000000000000a-005beb3839-101a-default</RequestId>
    <HostId>101a-default-default</HostId>
</Error>
```

Reply:
If the test fails in `master` as well as this branch, perhaps we could skip this test by wrapping it with something and using a feature flag to not run it on Ceph. See https://github.com/rusoto/rusoto/blob/master/integration_tests/tests/s3.rs#L70 for an example. 👍